Packs/OneLogin/Integrations/OneLoginEventCollector/OneLoginEventCollector.py

```diff
@@ -0,0 +1,367 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+from typing import Tuple
+from dateparser import parse
+from datetime import datetime, timedelta
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+VENDOR = "onelogin"
+PRODUCT = "iam"
+DEFAULT_LIMIT = 1000
+
+
+''' HELPER FUNCTIONS '''
+
+
+def arg_to_strtime(value: Any) -> Optional[str]:
+    if datetime_obj := arg_to_datetime(value):
+        return datetime_obj.strftime(DATE_FORMAT)
+    return None
+
+
+def increase_time_with_second(timestamp):
+    date_time_timestamp = parse(timestamp)
+    if not date_time_timestamp:
+        return timestamp
+    return datetime.strftime(date_time_timestamp + timedelta(seconds=1), DATE_FORMAT)
+
+
+def prepare_query_params(params: dict, last_run: dict = {}) -> dict:
+    """
+    Parses the given inputs into OneLogin Events API expected format.
+    """
+    query_params = {
+        'limit': arg_to_number(params.get('limit', DEFAULT_LIMIT)),
+        'since': arg_to_strtime(last_run.get('since') or params.get('since')),
+        'until': arg_to_strtime(params.get('until')),
+        'event_type_id': params.get('event_type_id'),
+        'after_cursor': last_run.get('after_cursor') or params.get('after_cursor')
+    }
+
+    return query_params
+
+
+def get_event_type_name(event_type_id, event_types):
+    """
+    Returns the event type name by the given event type ID.
+    """
+    if event_type_id in event_types:
+        return event_types[event_type_id]
+
+    return None
```

Comment:
will do the same
```suggestion
    return event_types.get(event_type_id)
```
don't really need a method for this though